src/python/WMComponent/WorkflowUpdater/WorkflowUpdaterPoller.py

```diff
@@ -61,10 +279,16 @@ def __init__(self, config):
 
         self.userCert = cert()
         self.userKey = ckey()
-        self.rucio = Rucio(acct=self.rucioAcct,
-                           hostUrl=self.rucioUrl,
-                           authUrl=self.rucioAuthUrl)
-                           # configDict={'logger': self.logger})
+        try:
```

Comment:
This was used in unit test, but I removed try/except block now and add optional argument to ctor to pass custom rucioClient (MockRucioApi in case of unit tests).